apps/expo-go/ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -2465,11 +2465,275 @@
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-Fabric\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-FabricImage\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-ImageManager\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-Mapbuffer\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-NativeModulesApple\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RCTAnimation\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RCTBlob\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RCTFabric\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RCTText\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RuntimeApple\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RuntimeCore\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-RuntimeHermes\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-graphics\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-jserrorhandler\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/React-nativeconfig\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/ReactCommon\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/RNSVG\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/RNScreens\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/libaom\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/libvmaf\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/libavif\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/libwebp\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-netinfo\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-pager-view\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-safe-area-context\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-segmented-control\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-slider\"",
+					"-isystem",
+					"\"${PODS_ROOT}/Headers/Public/react-native-webview\"",
+					"-iframework",
+					"\"${PODS_ROOT}/GoogleMaps/Base/Frameworks\"",
+					"-iframework",
+					"\"${PODS_ROOT}/GoogleMaps/Maps/Frameworks\"",
+					"-iframework",
+					"\"${PODS_XCFRAMEWORKS_BUILD_DIR}/GoogleMaps/Base\"",
+					"-iframework",
+					"\"${PODS_XCFRAMEWORKS_BUILD_DIR}/GoogleMaps/Maps\"",
+					"-iframework",
+					"\"${PODS_ROOT}/hermes-engine/destroot/Library/Frameworks/universal\"",
+					"-iframework",
+					"\"${PODS_XCFRAMEWORKS_BUILD_DIR}/hermes-engine/Pre-built\"",
+					"-iframework",
+					"\"${PODS_ROOT}/../vendored/unversioned/@shopify/react-native-skia/libs/ios\"",
+					"$(inherited)",
+					"-DFOLLY_NO_CONFIG",
+					"-DFOLLY_MOBILE=1",
+					"-DFOLLY_USE_LIBCPP=1",
+					"-Wno-comma",
+					"-Wno-shorten-64-to-32",
+					"-DREACT_NATIVE_MINOR_VERSION=74",
+					"-DREANIMATED_VERSION=3.6.0",
```

Comment:
please reset the file and edit again. i think this change was because you had expand the flags somehow.

Reply:
Fixed